docker-compose.yaml

```diff
@@ -9,11 +9,19 @@ services:
     #     REPO: Orcpub
     #     BRANCH: develop
     environment:
+      # Domain or IP address of the site. example.com or 192.168.77.1
+      SERVER-DOMAIN:''
+      # Protocol used. Either 'http://' or 'https://'
+      SERVER-PROTOCOL: 'https://'
```

Comment:
I think this is a bit misleading, or could be worded differently. What do think about making it a `SSL_ENABLED=true/false/1/0` flag and casting it to Bool in code?

Reply:
Fair enough. I'll update.